jupyterlab_gcedetails/auth_token_server/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "private": true,
```

Comment:
Not sure if there was a specific format to these, I just followed the one in the main folder